packages/cozy-mespapiers-lib/src/components/InstallAppFromIntent/InstallAppFromIntent.jsx

```diff
@@ -59,7 +80,7 @@ const InstallAppFromIntent = () => {
       }
       content={
         <Typography className="u-ta-center">
-          {t('InstallAppFromIntent.text')}
+          {t('InstallAppFromIntent.text', { appName: noteName })}
```

Comment:
OK let's remove my previous comment, you need to display this appName here 🤦 . 

It still not perfect since you do twice the same request. AppIcon and your fetch. 

Can't we pass the full object to AppIcon or pass a nullish fetchIcon to it in order to not make the. request twice?

Reply:
Après échange, il s'avère que nous avons bien 2 requêtes différentes :
- `/registry/<slug>` (faite via l'app)
- `/registry/<slug>/icon` (faite via AppIcon)

Rien d'anormal donc